README.md

```diff
@@ -81,6 +81,75 @@ expect(14).toEndWith("4");
 
 For a list of all matchers and extended documentation, please refer to the [API documentation](https://stackbuilders.github.io/assertive-ts/docs/build/).
 
+### Type Factory 🏭
+
+A great features of AssertiveTS is the type safety across the API. But, what should you do if you want to check if the value under test is of an specific type during runtime? The answer is simple, AssertiveTS provides a `.asType(TypeFactory)` method, where the [TypeFactory](https://stackbuilders.github.io/assertive-ts/docs/build/interfaces/TypeFactory.html) parameter let's you check for the specific type and let's you cast the assertion to a more specific one. To make things simpler, AssertiveTS provides [TypeFactories](https://stackbuilders.github.io/assertive-ts/docs/build/interfaces/StaticTypeFactories.html) for the basic types:
```

Comment:
@JoseLion the explanation here is really nice!! 
I found a couple of typos and a missing `(`.
```suggestion
A great feature of AssertiveTS is the type safety across the API. But, what should you do if you want to check if the value under test is of a specific type during runtime? The answer is simple, AssertiveTS provides a `.asType(TypeFactory)` method, where the [TypeFactory](https://stackbuilders.github.io/assertive-ts/docs/build/interfaces/TypeFactory.html) parameter lets you check for the specific type and lets you cast the assertion to a more specific one. To make things simpler, AssertiveTS provides [TypeFactories](https://stackbuilders.github.io/assertive-ts/docs/build/interfaces/StaticTypeFactories.html) for the basic types:
```